README.md

```diff
@@ -1 +1,48 @@
-# azure-sso-demo
\ No newline at end of file
+# azure-sso-demo
+
+This repository contains reference configurations that can create resourses on Azure using CLI and deploy cross-dc-rhsso-demo on Azure with all the required configurations.
+
+The final architecture looks like:
+
+![Architecture diagram](./scenario.jpg)
+
+
+## Prerequisites
```

Comment:
Can we separate each line (especially those with code snippets) into a purpose and code snippet.

For example

* Install Ansible Dependencies
```
ansible-galaxy collection install -r requirements.yml
```